defaults/main.yml

```diff
@@ -2,8 +2,84 @@
 keycloak_ver: '9.0.0'
 keycloak_mirror: https://downloads.jboss.org/keycloak
 
-keycloak_parent_install_dir: /usr/local
+keycloak_parent_install_dir: /opt
+keycloak_home: "{{ keycloak_parent_install_dir }}/keycloak"
 
+keycloak_java_packages: openjdk-11-jre
+
+keycloak_admin_username: admin
+
+keycloak_log_dir: "/var/log/keycloak"
+
+postgresql_driver_jar_mirror: https://jdbc.postgresql.org/download/
+postgresql_driver_jar_name: postgresql-42.2.12.jre6
+keycloak_postgresql_user: "keycloak"
+
+# SSL
+keycloak_ssl_src_dir: "/etc/letsencrypt/live/{{ keycloak_site_name }}"
+keycloak_ssl_cert_file: "fullchain.pem"
+keycloak_ssl_key_file: "privkey.pem"
+keycloak_certs_from_letsencrypt: true
+
+keycloak_http_site: "{{ keycloak_site_name }}-http"
+keycloak_https_site: "{{ keycloak_site_name }}-https"
```

Comment:
Rename and add `nginx` in the variable names. Also move the block to below #NGINX section